### ISSUE TITLE
How to build on current Ubuntu

### ISSUE THREAD

**Issue by jhit** [dlech/Keebuntu#57]
I would like to build this on a current Ubuntu 18.10.

Can you please give advise how to set up the build environment.

**Comment by dlech** [comment]
If you have the PPA setup, you can install the build dependencies via apt (something like `sudo apt build-dep keepass2-plugin-ubuntu`).

Otherwise, you can find the list here and manually install them: https://github.com/dlech/Keebuntu/blob/master/debian/control

Then `xbuild /property:Configuration=Release Keebuntu.sln` should build it.

**Comment by jhit** [comment]
Thank you @dlech .
I was able to update the reference to libimagemagick and compile and install.
Building only works with xbuild. msbuild fails with errors.
on the first run of keepass2 I got some crashes, but after 2 or 3 launches keepass runes without error now. Probably there were some old .net assemblies that caused the initial issues?

**Comment by dlech** [comment]
it would be helpful if you have the stack traces from the crashes. but if you can't make it crash again, we won't worry about it .

**Comment by jhit** [comment]
Ok. crashing still happens. Log attached.
[crash.log](https://github.com/dlech/Keebuntu/files/2911140/crash.log)

**Comment by dlech** [comment]
Hmm... the crash is not in the extension code. Although it hints that it has to do with bad character encoding. Are you perhaps using a translation in KeePass?

**Comment by jhit** [comment]
No, I use a clean install of the PPA for keepass2 without additional languages.
